tas-clients/resources/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tas-clients-serve
+  namespace: trusted-artifact-signer
+  labels:
+    app: trusted-artifact-signer-clients
+spec:
+  selector:
+    matchLabels:
+      app: trusted-artifact-signer-clients
+  template:
+    metadata:
+      labels:
+        app: trusted-artifact-signer-clients
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+      serviceAccountName: tas-clients
+      containers:
+      - name: tas-clients
+        image: quay.io/sallyom/tas-clients:httpd
```

Comment:
Chicken, meet egg. This should change to a rhtas image URI right?

Reply:
yes, we'll need to onboard the image, see the Containerfile above! Also, we still need to change out the 
`quay.io/sallyom/copy-secrets-job:latest` image in our values.yaml